Miniforge3/mambaforge_deprecation.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [[ "$GITHUB_ACTIONS" == "1" ]]; then
+    echo "::warning title=Mambaforge is now deprecated!::Future Miniforge releases will NOT build Mambaforge installers. We advise you switch to Miniforge at your earliest convenience."
+else
+    echo "!!!!!! Mambaforge is now deprecated !!!!!"
+    echo "Future Miniforge releases will NOT build Mambaforge installers."
+    echo "We advise you switch to Miniforge at your earliest convenience."
+    echo "Sleeping for 30s..."
+    sleep 30
```

Comment:
We should probably make an announcement before we do this. When do it though, we should include the sleep for GHA as well.

Reply:
https://github.com/conda-forge/conda-forge.github.io/pull/2242